packages/webpack-plugin/__tests__/index-test.js

```diff
@@ -157,24 +157,24 @@ describe('webpack-plugin-stylex', () => {
         var fadeAnimation = "xgnty7z-B";
         var styles = {
           foo: {
-            animationName: "xeuoslp",
-            display: "x78zum5",
-            marginInlineStart: "x1hm9lzh",
-            marginLeft: null,
-            marginRight: null,
-            marginTop: "xlrshdv",
-            height: "x1egiwwb",
-            ":hover_background": "x1oz5o6v",
-            ":hover_backgroundAttachment": null,
-            ":hover_backgroundClip": null,
-            ":hover_backgroundColor": null,
-            ":hover_backgroundImage": null,
-            ":hover_backgroundOrigin": null,
-            ":hover_backgroundPosition": null,
-            ":hover_backgroundPositionX": null,
-            ":hover_backgroundPositionY": null,
-            ":hover_backgroundRepeat": null,
-            ":hover_backgroundSize": null,
+            zba70f: "xeuoslp",
+            "1p303ag": "x78zum5",
```

Comment:
https://github.com/microsoft/griffel/blob/84fabade37aa190f0e80d2914f59e6db0ab936f7/packages/core/src/runtime/utils/hashPropertyKey.ts#L33C1-L38C4

This trick will help to avoid quotes if you want to save 2 more chars